src/Misc/layoutbin/en-US/strings.json

```diff
@@ -326,6 +326,7 @@
   "SavedSettings": "{0:u}: Settings Saved.",
   "ScanToolCapabilities": "Scanning for tool capabilities.",
   "SelfManageGitCreds": "You are in self manage git creds mode. Make sure your agent host machine can bypass any git authentication challenge.",
+  "ServerTarpit": "VSTS Agent is tarpitting by server, you may experience delay on console line post, job status report and task log upload.",
```

Comment:
The VSTS Agent is currently being throttled by the server. You may experience delays in console line output, job status reporting, and task log uploads.